crates/turborepo-ui/src/tui/app.rs

```diff
@@ -444,6 +448,10 @@ fn run_app_inner<B: Backend + std::io::Write>(
     let mut last_render = Instant::now();
     let mut callback = None;
     while let Some(event) = poll(app.input_options(), &receiver, last_render + FRAMERATE) {
+        // If we got a resize event, make sure to update ratatui backend.
+        if matches!(event, Event::Resize { .. }) {
+            terminal.autoresize()?;
+        }
```

Comment:
Does this need to get debounced at all? Since we're redrawing we probably want to wait until things stop changing for a few ms